plugins/jobtype/src/azkaban/jobtype/javautils/FileUtils.java

```diff
@@ -0,0 +1,47 @@
+package azkaban.jobtype.javautils;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+
+
+public class FileUtils {
+  private static Logger logger = Logger.getLogger(FileUtils.class);
+
+  /**
+   * Delete file or directory.
+   * (Apache FileUtils.deleteDirectory has a bug and is not working.)
```

Comment:
Could it be that you want to use FileUtils.forceDelete(), so that it would delete even if the folder is not empty?
http://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/FileUtils.html#forceDelete(java.io.File)